src/wp-login.php

```diff
@@ -199,6 +199,7 @@ function login_header( $title = null, $message = '', $wp_error = null ) {
 	?>
 	</head>
 	<body class="login no-js <?php echo esc_attr( implode( ' ', $classes ) ); ?>">
+	<main>
```

Comment:
Yeah, I was thinking that should change, too. I don't think we can anticipate the full scope, but footer and header are spaces that specifically should do something different. 

However, if we do that we need to wrap `login_header` in a `header` element, but it would not contain the h1. Given that the h1 is `screen-reader-text`, however, we can put it anywhere; it doesn't need to stay in the form.